Tests/RunnerPart2/StateFileTest.php

```diff
@@ -169,8 +169,7 @@ public function testPersistStateWithoutStateChange()
         $state = ['key' => 'fooBar'];
         $testLogger = new TestLogger();
         /** @var Client $sapiStub */
-        $clientWrapper = new ClientWrapper($sapiStub, null, null);
-        $clientWrapper->setBranchId('');
+        $clientWrapper = new ClientWrapper($sapiStub, null, null, ClientWrapper::BRANCH_MAIN);
```

Comment:
Jo fajn, to branchId jsem do konstruktoru přidával dodatečně a už jsem neměl energii to všude měnit :)

Reply:
Ja jsem to potreboval stejne vsechno projit abych nasel ty klienty s neinicializovanou branch. Tohle mi to dohledavani o kus zjednodusilo, protoze to pak byly v podstate vsechny vyskyty, kde ta branch v konstruktoru neni nastavena (je tam snad jen jedno nebo dve mista, kde byl potreba to nechat pres setter).